pkg/http/api.go

```diff
@@ -306,9 +306,14 @@ func CreateProject(host string, verifyTLS bool, apiKey string, name string, desc
 	return projectInfo, Error{}
 }
 
-// UpdateProject update a project
-func UpdateProject(host string, verifyTLS bool, apiKey string, project string, name string, description string) (models.ProjectInfo, Error) {
-	postBody := map[string]string{"name": name, "description": description}
+// UpdateProject update a project's name and (optional) description
+func UpdateProject(host string, verifyTLS bool, apiKey string, project string, name string, description ...string) (models.ProjectInfo, Error) {
+	postBody := map[string]string{"name": name}
+	if len(description) > 0 {
+		desc := description[0]
```

Comment:
super nit: `postBody["description"] := description[0]`

Reply:
Why is the description an array btw?

Reply:
We do this so that the description can be optional. We need to support `""` as a valid value, so we can't use that to signal whether a description was specified.